database.py

```diff
@@ -29,7 +32,7 @@ def get_db():
     
 
     def user_exists(self):
-        # is the user is the all_users table
+        """is the user is the all_users table"""
```

Comment:
Make sure the description makes sense. Also format it like the rest
Example:
"""
Something
"""